frontend/src/components/activePage/ActiveView/ActiveView.tsx

```diff
@@ -33,15 +33,30 @@ const ActiveView: FC<IActiveViewProps> = ({ ...props }) => {
   const [searchPopover, setSearchPopover] = useState(false);
   const [currentView, setCurrentView] = useState<WorkspaceRole>(
     managerView
-      ? (window.sessionStorage.getItem('prevViewActivePage') as WorkspaceRole)
+      ? (window.sessionStorage.getItem(
+          'prevViewActivePage'
```

Comment:
We'll have to make a utils for centrally managing the whole local/session storage. Using normal strings as keys in this way is not a big deal imo... Let's keep it like this now, you could maybe make a `const prevViewSSKey = 'prevViewActivePage'` and reference it in the various places in the meantime...